tests/uri.rs

```diff
@@ -24,6 +24,26 @@ macro_rules! req {
     )
 }
 
+macro_rules! res {
```

Comment:
Could probably be after these tests in `lib.rs`: https://github.com/seanmonstar/httparse/blob/5c385a9b1751f0734db24af731d1926e1d2bc731/src/lib.rs#L1118-L1136

I think these "uri" tests were ported from some parse test suite in Firefox or something...